4.7_pythagoras.py

```diff
@@ -0,0 +1,13 @@
+def pythagoras():
+    """gives any number combination forming the pythagoras triangle between 1 and 100"""
+    triangles = []
+    for i in range(1, 100):
+        for j in range(i, 100):
+            for k in range(j, 100):
+                if i**2+j**2 == k**2:
+                    triangle = (i, j, k)
+                    triangles.append(triangle)
+    return tuple(triangles)
+
+
+print(pythagoras())
```

Comment:
- kodlarınız çalışıyor. tebrikler

aşağıya farklı bir çözüm eklenmiştir.
def pythagorean(limit):
c = 0
m = 2 # for range(1,m --> m-1)

while c < limit:
    for n in range(1, m):
        a = (m * m) -(n * n)
        b = (2 * m * n)
        c = (m * m) + (n * n)
        if c > limit:
            break
        print(a, b, c)

    m = m + 1
limit = int(input("Enter a Limit : ")) # until limit it will write pythagorean tringles
pythagorean(limit)